src/gallia/services/uds/ecu.py

```diff
@@ -372,40 +372,19 @@ async def _tester_present_worker(self, interval: float) -> None:
         logger.debug("tester present worker started")
         while True:
             try:
-                async with self.mutex:
```

Comment:
Did other people in our team test this commit 6ac8ad5131989ab42645264edb11e7c7e0a73b34 on ECUs from a different manufacturer? I would like to avoid  that changes in team A break setups in team B.

Reply:
AFAIK this is tested on the vecu and ECUs from two manufacturers. But please be invited to test it further.

Reply:
@peckto will do some test runs on his setup.

Reply:
I understand it was now tested by @VeroSec @fkglr @peckto  and myself.

Reply:
> it was now tested …

Then let's move forward. ACK.